docs/sources/articles/configuring.md

```diff
@@ -96,3 +118,84 @@ These options :
 7. Verify that the `docker` daemon is running as specified wit the `ps` command.
 
 		$ ps aux | grep docker | grep -v grep
+
+### Logs
+
+By default logs for `Upstart` jobs are located in `/var/log/upstart` and the logs for `docker` daemon
+can be located at `/var/log/upstart/docker.log`
+
+    $ tail -f /var/log/upstart/docker.log
+    INFO[0000] Loading containers: done.
+    INFO[0000] docker daemon: 1.6.0 4749651; execdriver: native-0.2; graphdriver: aufs
```

Comment:
I don't have access to a vanilla Ubuntu install. Perhaps someone else following this PR is able to check?

it's a nit anyway, not a huge deal